solr/tests/common.py

```diff
@@ -0,0 +1,91 @@
+# (C) Datadog, Inc. 2020-present
+# All rights reserved
+# Licensed under Simplified BSD License (see LICENSE)
+import os
+
+from packaging import version
+
+from datadog_checks.dev import get_docker_hostname
+
+HOST = get_docker_hostname()
+SOLR_VERSION_RAW = os.environ['SOLR_VERSION']
+SOLR_VERSION = version.parse(SOLR_VERSION_RAW)
+
+JVM_METRICS = [
```

Comment:
note: can be replace later with `from datadog_checks.dev.jmx import JVM_E2E_METRICS` once this is merged: https://github.com/DataDog/integrations-core/pull/6486/files